package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "react-native-zoom-us",
-  "version": "4.0.4",
+  "version": "5.0.0",
   "description": "React-native bridge for ZoomUs SDK",
   "homepage": "https://github.com/mieszko4/react-native-zoom-us",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
~I guess that's fine, but I believe this will be a source of trouble for some people. We should support both - i.e. ts-build process is to be implemented (in future)~

> react-native works ok with lib .ts files,

cool, I did not know that

Reply:
Let's try this for now. If issue will appear, I will copy compilation from my libs.